node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
+  "systemParams": "darwin-x64-93",
+  "modulesFolders": [],
+  "flags": [],
+  "linkedModules": [],
+  "topLevelPatterns": [],
+  "lockfileEntries": {},
+  "files": [],
+  "artifacts": {}
+}
```

Comment:
You might want to remove this from the commit. I don't think you meant it to be in here.